proposals/statics-in-interfaces.md

```diff
@@ -127,38 +174,38 @@ These requirements need to be relaxed so that a restricted operand is allowed to
 
 ***Open question:** Should we relax this further so that the restricted operand can be of any type that derives from, or has one of some set of implicit conversions to `T`?*
 
-## Implementing static abstract members
+## Implementing static virtual members
 
-The rules for when a static member declaration in a class or struct is considered to implement a static abstract interface member, and for what requirements apply when it does, are the same as for instance members.
+The rules for when a static member declaration in a class or struct is considered to implement a static virtual interface member, and for what requirements apply when it does, are the same as for instance members.
 
 ***TBD:** There may be additional or different rules necessary here that we haven't yet thought of.*
 
-## Interface constraints with static abstract members
+## Interface constraints with static virtual members
 
 Today, when an interface `I` is used as a generic constraint, any type `T` with an implicit reference or boxing conversion to `I` is considered to satisfy that constraint.
 
-When `I` has static abstract members this needs to be further restricted so that `T` cannot itself be an interface.
+When `I` has static virtual members this needs to be further restricted so that `T` cannot itself be an interface.
 
 For instance:
 
 ``` c#
-// I and C as above
-void M<T>() where T : I<T> { ... }
+// I2<T> and C as above
+void M<T>() where T : I2<T> { ... }
 M<C>();  // Allowed: C is not an interface
-M<I<C>>(); // Disallowed: I is an interface
+M<I2<C>>(); // Disallowed: I is an interface
 ```
 
-## Accessing static abstract interface members
+## Accessing static virtual interface members
 
-A static abstract interface member `M` may be accessed on a type parameter `T` using the expression `T.M` when `T` is constrained by an interface `I` and `M` is an accessible static abstract member of `I`.
+A static virtual interface member `M` may be accessed on a type parameter `T` using the expression `T.M` when `T` is constrained (directly or indirectly) by an interface `I` and `M` is an accessible static virtual member of `I`.
```

Comment:
>A static virtual interface member `M` may be accessed on a type parameter `T` using the expression `T.M` [](start = 0, length = 104)

Can they be accessed without qualification inside default implementation?

Reply:
No. Only via a type parameter.